core/src/main/java/nl/nn/adapterframework/util/DateUtils.java

```diff
@@ -16,69 +16,88 @@
 package nl.nn.adapterframework.util;
 
 import java.text.ParseException;
-import java.text.ParsePosition;
-import java.text.SimpleDateFormat;
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
 import java.util.Calendar;
 import java.util.Date;
 
-import org.apache.logging.log4j.Logger;
+import lombok.extern.log4j.Log4j2;
+
 import org.apache.xmlbeans.GDate;
 
 /**
  * Utilities for formatting and parsing dates.
- * 
+ *
  * @author Johan Verrips IOS
  */
+@Log4j2
 public class DateUtils {
-	protected static Logger log = LogUtil.getLogger(DateUtils.class);
-
-	public static final String fullIsoFormat          = "yyyy-MM-dd'T'HH:mm:sszzz";
-	public static final String shortIsoFormat         = "yyyy-MM-dd";
-
-	public static final String FORMAT_FULL_GENERIC      = "yyyy-MM-dd HH:mm:ss.SSS";
-	public static final String FORMAT_MILLISECONDS	   ="######.###";
-	public static final String FORMAT_GENERICDATETIME  ="yyyy-MM-dd HH:mm:ss";
-	public static final String FORMAT_DATE             ="dd-MM-yy";
-	public static final String FORMAT_TIME_HMS         ="HH:mm:ss";
-
+	public static final String fullIsoFormat = "yyyy-MM-dd'T'HH:mm:sszzz";
+	public static final String shortIsoFormat = "yyyy-MM-dd";
+
+	public static final String FORMAT_FULL_GENERIC = "yyyy-MM-dd HH:mm:ss.SSS";
+	public static final String FORMAT_MILLISECONDS = "######.###";
+	public static final String FORMAT_GENERICDATETIME = "yyyy-MM-dd HH:mm:ss";
+	public static final String FORMAT_DATE = "dd-MM-yy";
+	public static final String FORMAT_TIME_HMS = "HH:mm:ss";
```

Comment:
Het is handig hier ook DateTimeFormatters voor te hebben, anders moet je die steeds opnieuw maken.